iscsi/iscsi.go

```diff
@@ -52,7 +52,7 @@ type Connector struct {
 	Multipath        bool         `json:"multipath"`
 	RetryCount       int32        `json:"retry_count"`
 	CheckInterval    int32        `json:"check_interval"`
-	DoDiscovery      bool         `json:"do_discovery"`
+	DoDiscoverydb    bool         `json:"do_discovery"`
```

Comment:
let's donot change the field name as lib users need to update the same.

Reply:
ok, reverted the name